packages/front-generator/src/test/fixtures/react-client/src/app/entity-cards/MpgFavoriteCarCards.tsx

```diff
@@ -1,23 +1,65 @@
 import * as React from "react";
 import { observer } from "mobx-react";
+
 import { FavoriteCar } from "cuba/entities/mpg$FavoriteCar";
-import { Card, Icon } from "antd";
-import { collection } from "@cuba-platform/react-core";
-import { EntityProperty } from "@cuba-platform/react-ui";
+import { Card } from "antd";
+import {
+  collection,
+  MainStoreInjected,
+  injectMainStore
+} from "@cuba-platform/react-core";
+import {
+  addPagingParams,
+  createPagingConfig,
+  defaultPagingConfig,
+  EntityProperty,
+  Paging,
+  setPagination,
+  Spinner
+} from "@cuba-platform/react-ui";
+import { action, IReactionDisposer, observable, reaction } from "mobx";
+import { PaginationConfig } from "antd/es/pagination";
+import { RouteComponentProps } from "react-router";
+
+type Props = MainStoreInjected & RouteComponentProps;
 
+@injectMainStore
 @observer
-export class MpgFavoriteCarCards extends React.Component {
+export class MpgFavoriteCarCards extends React.Component<Props> {
   dataCollection = collection<FavoriteCar>(FavoriteCar.NAME, {
     view: "favoriteCar-view",
-    sort: "-updateTs"
+    sort: "-updateTs",
+    loadImmediately: false
   });
+
+  @observable paginationConfig: PaginationConfig = { ...defaultPagingConfig };
+  reactionDisposer: IReactionDisposer;
   fields = ["notes", "car", "user"];
 
+  componentDidMount(): void {
+    // to disable paging config pass 'true' as disabled param in function below
+    this.paginationConfig = createPagingConfig(this.props.location.search);
+
+    this.reactionDisposer = reaction(
+      () => this.paginationConfig,
+      paginationConfig =>
+        setPagination(paginationConfig, this.dataCollection, true)
+    );
+    setPagination(this.paginationConfig, this.dataCollection, true);
+  }
+
+  componentWillUnmount() {
+    this.reactionDisposer();
+  }
+
   render() {
-    const { status, items } = this.dataCollection;
+    const { status, items, count } = this.dataCollection;
 
-    if (status === "LOADING") {
-      return <Icon type="spin" />;
+    if (
+      status === "LOADING" ||
+      this.props.mainStore?.isDataLoadedForEntityManagement !== true
```

Comment:
Do we really need to add this guard for non - editable screen?

Reply:
Not need here. We already have check inside EntityProperty.